### DISCUSSION TITLE
redux dead?

### DISCUSSION THREAD

**Discussion by Maximkalenkovich** [community/community#82992] [Other Feature Feedback, Questions, & Ideas]
I read the news that redux is officially no longer supported. If I understand correctly, he is already dead. What do you think will replace it?

**Comment by varunpats** [comment]
Hey @Maximkalenkovich, redux is not dead yet.

I think redux is widely used in the React ecosystem. There are alternatives, redux offers unique capabilities and a robust ecosystem that continues to attract developers.

In my opinion, redux is irreplaceable and more developers will exploit it. Long live Redux Toolkit!!

**Reply by Maximkalenkovich** [reply]
redux developer posted news that support for it is being discontinued
